playbooks/installation/index.adoc

```diff
@@ -57,25 +57,28 @@ We'll add to this file as we go.
 
 === Selecting the Version of OpenShift to Install
 
-Red Hat does major releases of OpenShift about once a quarter. As of this writing, the current major version is 3.5. Additionally, "dot releases" of OpenShift tend to be released every 4 to 6 weeks. The latest 3.5, for example, is 3.5.5.5. It's generally a good practice to decide on the major version of OpenShift to target and allow the install playbook to take the latest minor release of that major version. This will make sure you get all of the latest bug fixes and patches for that version.
+Red Hat does major releases of OpenShift about once a quarter. As of this writing, the current major version is 3.7. Additionally, "dot releases" of OpenShift tend to be released every 4 to 6 weeks. The latest 3.7, for example, is 3.7.14-5. It's generally a good practice to decide on the major version of OpenShift to target and allow the install playbook to take the latest minor release of that major version. This will make sure you get all of the latest bug fixes and patches for that version.
 
 You can set the version of OpenShift in your Ansible inventory file under the OSEv3:vars section.
 
 ----
 ...
 [OSEv3:vars]
-openshift_deployment_type=openshift-enterprise <1>
-openshift_release=v3.5 <2>
+ansible_ssh_user=root <1>
```

Comment:
believe the more current version of this var is `ansible_user`

Reply:
That looks to be right, but when I look at the 3.7 branch example hosts file it still shows ansible_ssh_user?
https://github.com/openshift/openshift-ansible/blob/release-3.7/inventory/byo/hosts.example

Reply:
both are correct, but `ansible_ssh_user` will be deprecated [1]

Ansible 2.0 has deprecated the “ssh” from ansible_ssh_user, ansible_ssh_host, and ansible_ssh_port to become ansible_user, ansible_host, and ansible_port. If you are using a version of Ansible prior to 2.0, you should continue using the older style variables (ansible_ssh_*). These shorter variables are ignored, without warning, in older versions of Ansible.

[1] http://docs.ansible.com/ansible/latest/intro_inventory.html

Reply:
Updated..